client/test/testdata/fake-hcdata.js

```diff
@@ -0,0 +1,1974 @@
+// TODO: use typescript to ensure that the hardcoded fake data conforms
```

Comment:
we should replace it by including a fpkm gz file to termdbtest, so that `query.geneExpress` will work for termdbtest